include/swift/ClangImporter/ClangImporter.h

```diff
@@ -218,6 +223,14 @@ class ClangImporter final : public ClangModuleLoader {
   std::string getBridgingHeaderContents(StringRef headerPath, off_t &fileSize,
                                         time_t &fileModTime);
 
+  /// Makes a temporary replica of the ClangImporter's CompilerInstance, reads
+  /// an Objective-C header file into the replica and emits a PCH file of its
```

Comment:
These docblocks are fantastic!

nit-pick: Sorry to harp on about acronyms, but I noticed "PCH" isn't in `docs/Lexicon.rst`, either. I think that some Swift contributors might not be immediately familiar with the concept of a precompiled header. Since this pull request is the first use of "PCH" in a docblock in this codebase, I'd suggest adding the acronym to the Lexicon as well.